config/locales/en.yml

```diff
@@ -30,7 +30,14 @@ en:
 
       Princeton users with copyright questions or concerns should refer to <a href="http://www.princeton.edu/copyright/">http://www.princeton.edu/copyright/</a>.
   simple_form:
+    labels:
+      defaults:
+        ocr_language: "OCR Language"
     options:
+      defaults:
+        ocr_language:
+          eng: "English"
```

Comment:
Do we want to include the other languages we expect to handle?  And do we want to include more info (e.g., the ISO 639-2 codes where different)?  This might let us use a single language field for both the language of the resource and the OCR language to use.